charts/metrics-server/templates/configmaps-nanny.yaml

```diff
@@ -13,5 +13,6 @@ data:
     baseCPU: {{ .Values.addonResizer.nanny.cpu }}
     cpuPerNode: {{ .Values.addonResizer.nanny.extraCpu }}
     baseMemory: {{ .Values.addonResizer.nanny.memory }}
+    baseStorage: {{ .Values.addonResizer.nanny.storage }}
```

Comment:
Yes, that is missing. I think that means we shouldn't parse the following:
`Base:         resource.MustParse(*baseStorage),` 

Ref: https://github.com/kubernetes/autoscaler/blob/cf606a1400d6e421d32349e25115b508382111ff/addon-resizer/main.go#L163C51-L164C1

Reply:
I'm not sure I understand your reply. I think you'll have to conditionally set command-line arguments for storage and extra-storage, because those don't seem to be supported by the config file. Might want to confirm with sig-autoscaling, as they are the owners for [addon-resizer](https://github.com/kubernetes/autoscaler/tree/master/addon-resizer).